lib/00_core.sh

```diff
@@ -196,6 +196,9 @@ _translate_noconfirm() {
   "zypper") _opt="--no-confirm";;
   "pkgng")  _opt="-y";;
   "tazpkg") _opt="--auto";;
+  # apk is supposed to run in non-interactive mode by default (FIXME)
```

Comment:
Hmm, this make me wonder that should we return `-i` in case of missing `--no-confirm`? That should fix the `FIXME`.

Reply:
Interesting... I will need to read `apk` again

Reply:
@cuonglm I think there are 3 things

* When user specifies `--no-confirm`, we need to return empty string (instead of an error)
* When user specifies `<let's-go-interactive>`, we would return `-i`
* When user doesn't specify any option, we would assume it's the first case

So this Pr is more the 1st and 3rd cases, isn't it? I can't find an doc regard that feature of `apk`, that what I meant for the FIXME item.

Reply:
> When user doesn't specify any option, we would assume it's the first case

I think this means `--no-confirm` is not specified, we should run interactively 🤔

Reply:
@cuonglm I agreed with you, but then I think twice. We don't try to fix the behavior of the script (interactive vs non-interactive). We let `apk` do that, and if possible, we give another option for `apk` (in the recent commit, I also add `_TOPT`, but this currently doesn't work because `-i` option has a different mean in the original `pacman`, -- I would have to refine the argument parsing step).

What we need to fix right now, is to allow our script to continue to run, instead of generating an error as seen in https://github.com/icy/pacapt/issues/150.

What do you think?

PS: I would have had tests for this apk. I'm also using them a lot ;)

Reply:
Thanks @cuonglm for your approval. I will create a follow-up ticket to address the interactive option for `apk`.